### PR TITLE
Add an email to SR2025 secondary contacts

### PR DESCRIPTION
**Pull request by PeterJCLaw**
## Summary

So that they know what's going on. Also means we'll find out sooner than later if their email address doesn't work.

This also includes links to our docs around the kit loan and the team supervisor responsibilities.

Would be great to merge https://github.com/srobo/docs/pull/614 before we send this, but it's not a blocker.

Fixes https://github.com/srobo/tasks/issues/1467

## Checklist

- [x] Has front-matter (`to` and `subject`)
- [x] Compared to a similar email from the previous year
- [x] Ensured there's enough context for a rookie team to understand

### CODE REVIEW

File: SR2025/2024-09-18-secondary-contact-info.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+As you're hopefully aware from $TODO_PRIMARY_CONTACT_FULL_NAME,
+[Student Robotics][student-robotics] is an exciting competition for
+16-19 year-olds which a team from $TODO_ORGANISATION is taking part in.
+
+$TODO_PRIMARY_CONTACT_GIVEN_NAME gave us your details as a secondary contact at
+$TODO_ORGANISATION in the unlikely case that we are unable to contact them.
+Please speak to them in the first instance if you have any questions.
+
+As [team supervisor][team-supervisor] $TODO_PRIMARY_CONTACT_GIVEN_NAME is
```
Comment (samjmartin04) [inline SR2025/2024-09-18-secondary-contact-info.md:16]:
```suggestion
As [team supervisor][team-supervisor], $TODO_PRIMARY_CONTACT_GIVEN_NAME is
```

File: SR2025/2024-09-18-secondary-contact-info.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+responsible for your competitors as well as the [kit][kit] which we loan to each
```
Comment (samjmartin04) [inline SR2025/2024-09-18-secondary-contact-info.md:17]:
```suggestion
responsible for your competitors as well, as the [kit][kit] which we loan to each
```

**Review by samjmartin04 — APPROVED** [review]

File: SR2025/2024-09-18-secondary-contact-info.md (hunk 2 above)
Comment (PeterJCLaw) [reply SR2025/2024-09-18-secondary-contact-info.md:17]: Not sure what you're suggesting here? This breaks up the "as well as" phrase -- is that intentional?

File: SR2025/2024-09-18-secondary-contact-info.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+[Student Robotics][student-robotics] is an exciting competition for
+16-19 year-olds which a team from $TODO_ORGANISATION is taking part in.
+
+$TODO_PRIMARY_CONTACT_GIVEN_NAME gave us your details as a secondary contact at
+$TODO_ORGANISATION in the unlikely case that we are unable to contact them.
+Please speak to them in the first instance if you have any questions.
+
+As [team supervisor][team-supervisor], $TODO_PRIMARY_CONTACT_GIVEN_NAME is
+responsible for your competitors as well as the [kit][kit] which we loan to each
+team. Our communication with your team will also all be sent to them.
```
Comment (PeterJCLaw) [inline SR2025/2024-09-18-secondary-contact-info.md:18]: There's two things I'm trying to convey here:
- awareness of the kit loan
- awareness that if the primary contact disappears that we'll chase the secondary for the kit and for looking after the competitors

I almost want to make it clear that the secondary is thus also responsible for the kit, but I'm not sure if there's a way to do that without it getting really complicated.

File: SR2025/2024-09-18-secondary-contact-info.md (hunk 2 above)
Comment (samjmartin04) [reply SR2025/2024-09-18-secondary-contact-info.md:17]: Was trying to improve the flow of the sentence, but if we want to emphasise the two responsibilities, then we should keep it how it was

**Review by sedders123 — DISMISSED** [review]

File: SR2025/2024-09-18-secondary-contact-info.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+---
+to: Student Robotics 2025 secondary contacts, CC primary contacts
```
Comment (PeterJCLaw) [inline SR2025/2024-09-18-secondary-contact-info.md:2]: Hrm, just spotted that Gmail's mail merge doesn't support CC :( Not sure how best to do this. Might need to script something ourselves.

Comment (PeterJCLaw) [reply SR2025/2024-09-18-secondary-contact-info.md]: Have scripted this and sent it out. Added the script here in case it's useful in future.

**Comment by PeterJCLaw** [comment]
Going to merge since this was sent.

**Pull request merged**